packages/component-submission/server/use-cases/submitSurveyUseCase.test.js

```diff
@@ -0,0 +1,62 @@
+const { makeResponseObject } = require('./submitSurveyUseCase')
+
+describe('submitSurveyUseCase', () => {
+  it('transforms the input type into the database type', () => {
+    const input = [
+      {
+        questionId: 'question1',
+        text: "What's your favourite kind of sock?",
+        answer: 'answer1',
+      },
+      {
+        questionId: 'question2',
+        text: 'WHY ARE YOU RUNNING?',
+        answer: 'answer2',
+      },
+      {
+        questionId: 'question3',
+        text: 'What?',
+        answer: 'answer3',
+      },
+      {
+        // There shouldn't be duplicate question answers, but if there is, what happens?
+        questionId: 'question3',
+        text: 'What?',
+        answer: 'answer3',
+      },
+    ]
+
+    const result = makeResponseObject(input)
```

Comment:
This is not really testing the use case object, only the makeResponseObject method which is an implementation detail.

The test should be testing the actual Use Case object. See getSubmissionUseCase.test.js file for example